.github/workflows/ci.yml

```
@@ -5,16 +5,16 @@ on: [push, pull_request]
 jobs:
   test:
     name: OTP ${{matrix.otp}} / Elixir ${{matrix.elixir}}
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     env:
       MIX_ENV: test
     strategy:
       matrix:
-        elixir: [ 1.15.4 ]
+        elixir: [ 1.18.0 ]
```

En el resto de casos hemos puesto 1.18.4

```suggestion
        elixir: [ 1.18.4 ]
```